Aztec/Classes/Converters/NSAttributedStringToNodes.swift

```diff
@@ -43,6 +42,31 @@ class NSAttributedStringToNodes: Converter {
     }
 
 
+    /// Converts a Substring, defined by it's paragraphRange and enclosingRange, into a collection of Nodes,
+    /// representing the internal HTML Entities.
+    ///
+    /// Note:
+    /// Whenever the paragraph defined by the paragraphRange is zero, we'll proceed extracting all of the attributes
+    /// present in the Enclosing Range. Otherwise, we might loose data!
+    ///
+    /// - Parameter:
+    ///     - attrString: Reference to the document to be converted.
+    ///     - paragraphRange: Defines the Paragraph Range to be converted into Nodes.
+    ///     - enclosingRange: Defines the Paragraph's Enclosing Range (containing newline characters).
+    ///
+    /// - Returns: Array of Node instances.
+    ///
+    private func createNodes(fromString attrString: NSAttributedString, paragraphRange: NSRange, enclosingRange: NSRange) -> [Node] {
```

Comment:
Consider renaming `fromString attrString:` to `fromParagraph paragraph:` since it focuses more on the logical significance on the string, and less on its type.